.github/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Release 0.2.0-dev
```

Comment:
We haven't been very good at updating the changelog in this repo :stuck_out_tongue_closed_eyes: 

```suggestion
# Release 0.5.0-dev
```